src/components/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Header() {
+  return (
+    <header className="background-style">
+      <nav>
+        <h1>Math Magicians</h1>
+        <ul id="desktop-nav">
+          <li><Link to="/">Home</Link></li>
+          <li><Link to="/calculator">Calculator</Link></li>
+          <li><Link to="/quote">Quote</Link></li>
```

Comment:
- [ ] [OPTIONAL] Good job creating a standalone component for the header. :+1: React Router provides a special type of link called `NavLink` which is intended to be used in navigation. It makes applying styles on the currently active link easier. You can check out React Router [documentation](https://reactrouter.com/en/main/components/nav-link) for more details. 😎️